index.html

```diff
@@ -64,7 +71,7 @@ <h3>LET'S CONNECT</h3>
 
             <!-- Works Section -->
             <section class="works-section">
```

Comment:
- [ ] When you click the `Portfolio` nav link, it should take you to the works section. It is also good to add a heading such as `Works` or `Projects` to this section.